backend-diagram-converter/core/src/main/resources/message-templates.properties

```diff
@@ -187,10 +203,20 @@ inclusive-gateway-join.severity=WARNING
 conditional-flow.message=A Conditional flow is not supported.
 conditional-flow.severity=WARNING
 #
+internal-script.message=Script is transformed to Zeebe script.
+internal-script.severity=INFO
+#
 # BPMN attribute
 #
 script-format.message=Script format '{{ scriptFormat }}' was set to header '{{ headerName }}'. Please review.
 script-format.severity=REVIEW
 #
 script-format-missing.message=Script format could not be found. Please review.
-script-format-missing.severity=TASK
\ No newline at end of file
+script-format-missing.severity=TASK
+#
+escalation-code.message=Escalation code is transformed from '{{ oldCode }}' to '{{ newCode }}'. Please review.
+escalation-code.severity=REVIEW
```

Comment:
Is this a value or an expression? If it is a value, INFO would be OK.

Reply:
Escalation code is an expression 👍